controllers/usersignup/usersignup_controller.go

```diff
@@ -72,6 +73,7 @@ type Reconciler struct {
 	Scheme         *runtime.Scheme
 	SegmentClient  *segment.Client
 	ClusterManager *capacity.ClusterManager
+	PublicViewer   commonconfig.PublicViewerConfig
```

Comment:
Is this going to be used in follow-up PRs ? 
It doesn't seem to be used right now.

Reply:
Seems something that I forgot to remove 😄 thanks for noticing it!

Reply:
removed in 84e431dcac88feaec16665874fb880c19c4a397f